website/client/src/components/shops/quests/index.vue

```diff
@@ -209,7 +209,6 @@
           <!-- eslint-disable vue/no-use-v-if-with-v-for, max-len -->
           <div
             v-for="(items, key) in getGrouped(questItems(category, selectedSortItemsBy,searchTextThrottled, hideLocked, hidePinned))"
-            v-if="key !== 'questGroupEarnable'"
```

Comment:
my understanding from the issue is that these quests will appear but won't be purchasable, do we need to show anything (text, ..) to explain this?

Reply:
The intent is for the quests to be purchasable!